app/(protected)/(tabs)/card/user_info_mobile.tsx

```diff
@@ -0,0 +1,242 @@
+import * as WebBrowser from "expo-web-browser";
+import React, { useState } from "react";
+import { Linking, Pressable, TextInput, View } from "react-native";
+import { z } from "zod";
+
+import { Button } from "@/components/ui/button";
+import { Text } from "@/components/ui/text";
+import { path } from "@/constants/path";
+import { KycStatus } from "@/lib/types";
+import { useRouter } from "expo-router";
+import { createKycLink } from "@/lib/api";
+
+// Header Component
+function UserInfoHeader() {
+  return (
+    <Text className="text-base text-white/70 text-center">
+      We need some basic information to get started with your card activation
+    </Text>
+  );
+}
+
+// Form Component
+interface UserInfoFormProps {
+  fullName: string;
+  setFullName: (value: string) => void;
+  email: string;
+  setEmail: (value: string) => void;
+}
+
+function UserInfoForm({
+  fullName,
+  setFullName,
+  email,
+  setEmail,
+}: UserInfoFormProps) {
```

Comment:
@SametSahin10 Please use a proper from validation using `react-hook-form`, `@hookform/resolvers` & `zod`. There are examples in the `app/(protected)/(tabs)/deposit.tsx` & `components/Withdraw/index.tsx`